initial-reverse/reverse.cc

```diff
@@ -0,0 +1,126 @@
+/*
+ * =====================================================================================
+ *
+ *       Filename:  reverse.cc
+ *
+ *    Description:  Reverses the lines of a file
+ *
+ *        Version:  1.0
+ *        Created:  07/04/2020 14:49:10
+ *       Revision:  none
+ *       Compiler:  g++
+ *
+ *         Author:  Brian Li,
+ *   Organization:
+ *
+ * =====================================================================================
+ */
+#include <iostream>
+#include <stdexcept>
+#include <sys/stat.h>
+
+template <class T> class Vector {
+  // Backing array
```

Comment:
put it in private section. private variables should have underscore at the end of their names. private section should also come after the public section